src/segmentation/train.py

```diff
@@ -75,9 +83,17 @@ def init_from_valid_args(cls, args):
 
     trainer.fit(model, datamodule=fibers)
 
-    torch.save(model.state_dict(), "trained_model.pth")
+    torch.onnx.export(
+        model=model,
+        args=torch.randn(1, 2, 256, 256),
+        f=cfg.model_path,
+        export_params=True,
+        verbose=True,
+        input_names=["input"],
+        output_names=["segmentation"],
+    )
 
     trained_model_artifact = wandb.Artifact("model", type="model", metadata=cfg)
```

Comment:
Same thing about the name of the model here. Can it please be named `seg_model`?